From 0da23c01c728cda0f6bdbf99f7f3f478c0d2e445 Mon Sep 17 00:00:00 2001
From: "Radu Pirea (NXP OSS)" <radu-nicolae.pirea@oss.nxp.com>
Date: Tue, 15 Feb 2022 11:30:38 +0200
Subject: [PATCH 32/43] configs: s32: set default console to ttyLF0

CONFIG_FSL_LINFLEX_MODULE has been removed.
Without CONFIG_FSL_LINFLEX_MODULE set to 0 console variable will be set
to ttyLFCONFIG_FSL_LINFLEX_MODULE.

Issue: ALB-8504

Upstream-Status: Pending 

Signed-off-by: Radu Pirea (NXP OSS) <radu-nicolae.pirea@oss.nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/s32.h b/include/configs/s32.h
index 522712a1d1..97918a758d 100644
--- a/include/configs/s32.h
+++ b/include/configs/s32.h
@@ -300,7 +300,7 @@
 	"boot_mtd=" __stringify(BOOT_MTD) "\0" \
 	"image=" __stringify(IMAGE_NAME) "\0" \
 	"ramdisk=" __stringify(RAMDISK_NAME) "\0"\
-	"console=ttyLF" __stringify(CONFIG_FSL_LINFLEX_MODULE) "\0" \
+	"console=ttyLF0\0" \
 	"fdt_high=0xa0000000\0" \
 	"initrd_high=" __stringify(INITRD_HIGH) "\0" \
 	"fdt_file="  __stringify(FDT_FILE) "\0" \
-- 
2.17.1

